content/docs/guide.md

```diff
@@ -68,10 +68,27 @@ If none of the above work for you, try running it in a container
 from our Fedora based image. It contains packit installed from `main` branch,
 i.e. the same you'd get by pip installing from Github.
 
-    $ podman run -ti --rm -v $PWD:/src --security-opt label=disable quay.io/packit/packit bash
+    $ podman run -ti --rm -v $PWD:/src:z quay.io/packit/packit bash
     $ packit
     Usage: packit [OPTIONS] COMMAND [ARGS]...
 
+Based upon the command you need to performer you should configure your container with the needed 
+secrets and configuration files like in the following examples:
+
+#### packit copr-build
+
+    $ podman run -ti --rm -v ~/.config/copr:/root/.config/copr:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ packit copr-build
+
+#### packit build
+
+    $ podman run -ti --rm -v ~/.ssh/:/root/.ssh:z -v $PWD:/src:z quay.io/packit/packit bash
+    $ dnf install krb5-workstation
```

Comment:
btw, what about adding this package to the image? Do you know how huge is this with all the dependencies?

Reply:
I thought the same, multiple times while waiting for the command to complete! For me is an improvement. If you do not have any concern with the image being more heavy.

Reply:
Looking on its size I realized it is already installed. So I can remove that line!

Reply:
Aha..;) So it just built the cache and installed nothing. Nice!